### PR TITLE
doc/releases: write a note about the "invalid regex" mgr bug

### PR DESCRIPTION
**Pull request by ljflores**
Signed-off-by: Laura Flores <lflores@redhat.com>

## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "pacific"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [x] Doc update (no ticket needed)
  - [ ] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [x] Updates relevant documentation
  - [ ] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
</details>

### CODE REVIEW

**Review by tchaikov — APPROVED** [review]

**Review by batrick — APPROVED** [review]

**Review by anthonyeleven — APPROVED** [review]

File: doc/releases/quincy.rst [hunk 1]
```diff
@@ -5,6 +5,26 @@ Quincy
...
 
+v17.2.3 Quincy
+==============
+
+This is a hotfix release that addresses a libcephsqlite crash in the mgr.
+
+Notable Changes
+---------------
+* A libcephsqlite bug that caused the mgr to crash repeatedly and die is now
+  fixed. The bug was exposed due to 17.2.2 using gcc 8.5.0-14, which contains
```
Comment (djgalloway) [inline doc/releases/quincy.rst:16]:
```suggestion
  fixed. The bug was exposed due to 17.2.2 being built with gcc 8.5.0-14, which contains
```

File: doc/releases/quincy.rst [hunk 2]
```diff
@@ -5,6 +5,26 @@ Quincy
...
+  a new patch to check for invalid regex. 17.2.1 was using gcc 8.5.0-13, which
```
Comment (djgalloway) [inline doc/releases/quincy.rst:17]:
```suggestion
  a new patch to check for invalid regex. 17.2.1 was built using gcc 8.5.0-13, which
```

File: doc/releases/quincy.rst (hunk 1 above)
Comment (ljflores) [reply doc/releases/quincy.rst]: Fixed

File: doc/releases/quincy.rst (hunk 2 above)
Comment (ljflores) [reply doc/releases/quincy.rst]: Fixed

**Comment by ljflores** [comment]
Rendered page: https://ceph--47343.org.readthedocs.build/en/47343/releases/quincy/

**Review by neha-ojha — APPROVED** [review]

**Pull request merged**